README.md

```diff
@@ -238,4 +238,47 @@ How to check status and read connection metrics:
     curl --cacert test-keys/root.crt https://localhost:6060/_metrics
```

Comment:
It turns out fine in the Markdown rendering, since the indented block is a code block.